src/pages/dar_application/ResearcherInfo.js

```diff
@@ -67,7 +59,7 @@ export default function ResearcherInfo(props) {
               className: 'checkbox-inline rp-checkbox',
               disabled: !isNil(props.darCode),
               checked: checkCollaborator,
-              onChange: (e) => formStateChange(setCheckCollaborator, 'checked', e)
+              onChange: (e) => props.formStateChange(setCheckCollaborator, 'checked', e)
```

Comment:
One bit of code structure that I struggle with in our existing code is that referring to props directly, spread out through the html, is confusing. I like having all of the required props destructured into constants in one place, usually at the top of the render method, so as a dev, I can see all of the component requirements in one place. In a typical component, that's usually in the constructor. In a functional world, we should choose someplace that makes sense for us going forward.